tsl/test/isolation/expected/continuous_aggs_insert.out

```diff
@@ -55,13 +48,11 @@ count
 step Refresh: REFRESH MATERIALIZED VIEW continuous_view; <waiting ...>
 step Sc: COMMIT;
 step UnlockCompleted: ROLLBACK;
-INFO:  materializing continuous aggregate public.continuous_view: nothing to invalidate, new range up to 15
```

Comment:
I think we want to keep the messages in the isolation test as well.

Reply:
To expand on this, part of what these tests check is when materialization can see concurrent INSERTs, and these messages are the only way we have to determine if the INSERTs were seen. (We use integer times for this, so the test _should_ be stable...)